task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -189,6 +189,8 @@ spec:
       fi
 
       rsync -ra $(workspaces.source.path)/ "$SSH_HOST:$BUILD_DIR/workspaces/source/"
+      rsync -ra /entitlement/ "$SSH_HOST:$BUILD_DIR/volumes/etc-pki-entitlement/"
+      rsync -ra /mnt/trusted-ca/ "$SSH_HOST:$BUILD_DIR/volumes/trusted-ca/"
```

Comment:
will it still work if `/mnt/trusted-ca` doesn't exist? mounting the configmap to that directory is optional and only happens with the configmap with the ca bundle is configured in the namespace.

Reply:
Sorry, I forgot to respond to this. I tested it locally and it seemed to work fine.

Reply:
thanks for checking